shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/metadata/model/PipelineTableMetaDataTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.shardingsphere.data.pipeline.core.metadata.model;
+
```

Comment:
License header should be added at the beginning of java code. You could refer to other source code.

And it's better to run unit test before submiting PR. How to do it?
1. In terminal, cd to `shardingsphere-test/shardingsphere-pipeline-test` directory (assume it's in shardingsphere project root)
2. Execute command: `mvn clean install -Dmaven.javadoc.skip=true -Prelease`, fix error and run it again until it pass.

Reply:
Added the license header.

Earlier I was trying to run ```./mvnw clean install `-Dmaven`.`javadoc`.skip=true -Prelease``` at project root since I am running the project on Windows. I ran into several errors with several tests of the  `shardingsphere-sql-parser-engine`  module failing. 

I will try running the unit tests again in the `shardingsphere-pipeline-test` directory.